packager/util/config.go

```diff
@@ -66,10 +66,17 @@ type ACMEConfig struct {
 }
 
 type ACMEServerConfig struct {
-	DiscoURL   string // ACME Directory Resource URL
-	AccountURL string // ACME Account URL. If non-empty, we
-	// will auto-renew cert via ACME.
-	EmailAddress string // Email address registered with ACME CA.
+	// ACME Directory Resource URL
+	AccountURL string
+	// ACME Account URL. If non-empty, we will auto-renew cert via ACME.
+	DiscoURL string
+	// Email address registered with ACME CA.
+	EmailAddress string
+	// Key Identifier from ACME CA. Used for External Account Binding.
+	EABKid string
+	// MAC Key from ACME CA. Used for External Account Binding. Should be in
+	// Base64 URL Encoding without padding format.
+	EABHmac string
```

Comment:
Add documentation in amppkg.example.toml? (Or make sure it's on your TODO list.)

Reply:
yeah this will be in the next PR.